server/urls.py

```diff
@@ -24,6 +21,9 @@
     url(r'^logout/$', server.views.logout, name='logout'),
     url(r'^login/?', server.views.login, name='login'),
 
+    # user urls
+    url(r'^(?P<name>[a-zA-Z0-9]+)', server.views.user, name='user'),
```

Comment:
I would probably move this to the very bottom, certainly below `admin/` (though in fact we might just want to delete that since we're not using it, another story)

Reply:
Sounds good - I'll move it to the very bottom. What happens when a user creates an account named `login` or `notebooks`?

Reply:
At the moment this isn't possible, because we use github for auth and github doesn't allow such names. If we ever allow the creation of usernames we should probably restrict the allowed set the same way github does:

![image](https://user-images.githubusercontent.com/20569/45110613-c8b99a00-b110-11e8-85f8-eb97cd8494fa.png)

Reply:
Yeah, I'm assuming we'll allow the creation of user names down the line. I'm sure there is an easy django way to handle this :)